app/models/vaccine.rb

```diff
@@ -21,15 +22,15 @@ class Vaccine < ApplicationRecord
   # --- FIELD VALIDATION --- #
   validates :group_name, inclusion: {
     # This method syntax is necessary for the getter method to be in scope
-    in: ->(_vaccine) { group_name_options },
+    in: ->(_vaccine) { [*group_name_options, nil] },
```

Comment:
Since this makes `nil` an acceptable value, then this test `bundle exec rails test -n test_validates_group_name` needs to be updated to reflect that

Reply:
https://github.com/SaraAlert/SaraAlert/pull/991/files#diff-55f69eadeff0a340b9e0c84bf77a5e21f1745e101319cc8603ab2c8b9f4f107eR19